### PR TITLE
feat: add a couple of languages

### PR DESCRIPTION
**Pull request by zacanger**
Adds Python and TypeScript, adds a few missing keywords in JS, and also removes duplicate unused copy of jQuery.

### CODE REVIEW

**Review by TheTechRobo — APPROVED** [review]
Seems OK

**Comment by zacanger** [comment]
Seventeen seconds after I put in the PR :laughing: you're fast

File: static/css/jquery.js [hunk 1]
```diff
@@ -1,4 +0,0 @@
-/*! jQuery v2.2.4 | (c) jQuery Foundation | jquery.org/license */
```
Comment (medvednikov) [inline static/css/jquery.js:1]: no need to delete jquery :)

Comment (spytheman) [reply static/css/jquery.js:1]: There is already a jquery copy in https://github.com/vlang/gitly/blob/master/static/js/jquery.js
The one in css/jquery.js is redundant.

Comment (medvednikov) [reply static/css/jquery.js:1]: indeed.

**Pull request merged**